enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/rest/ClientErrorResponseDto.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.provider.rest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import java.util.Objects;
+
+/**
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@JsonTypeName("ClientErrorResponse")
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen")
+class ClientErrorResponseDto {
+
+    private String message;
+    private String errorUUID;
```

Comment:
I would use simple `errorId`, the UUID is not needed information imho?

Reply:
It is generated based on swagger from BSC. But yes, that we may make it better in product and change it in downstream. Or better, I do not deserialize error jsons at all.